Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxRegEx/TextBoxRegex.cs

```diff
@@ -54,7 +54,15 @@ private static void TextBoxRegexPropertyOnChange(DependencyObject sender, Depend
         private static void Textbox_TextChanged(object sender, TextChangedEventArgs e)
         {
             var textbox = (TextBox)sender;
-            ValidateTextBox(textbox, false);
+            var validationMode = (ValidationMode)textbox.GetValue(ValidationModeProperty);
```

Comment:
Remove the if and pass the value of the if statement to the second parameter
`ValidateTextBox(textbox, (validationMode == ValidationMode.Instantly || validationMode == ValidationMode.InstantlyNumber));`

Reply:
Done.